contrib/win32/openssh/install-sshd.ps1

```diff
@@ -84,6 +85,19 @@ if (Test-Path $moduliPath -PathType Leaf)
     Repair-ModuliFilePermission -FilePath $moduliPath @psBoundParameters -confirm:$false
 }
 
+#If %programData%/ssh folder already exists, fix permissions
+$sshProgDataPath = Join-Path $env:ProgramData "ssh"
+if (Test-Path $sshProgDataPath)
+{
+    $sshProgDataAcl=Get-Acl $sshProgDataPath
+    # Folder permission is FullAccess to System and Builtin/Admins and read only access to Authenticated users
+    $sshProgDataAcl.SetSecurityDescriptorSddlForm("O:BAD:PAI(A;OICI;FA;;;SY)(A;OICI;FA;;;BA)(A;OICI;0x1200a9;;;AU)")
+    Set-Acl $sshProgDataPath $sshProgDataAcl
+    # key files should only allow FullAccess to System and Builtin/Admins
+    $sshProgDataAcl.SetSecurityDescriptorSddlForm("O:BAD:PAI(A;OICI;FA;;;SY)(A;OICI;FA;;;BA)")
```

Comment:
I think Windows will automatically fix it, but inheritance flags OI / CI do not really belong on non-container objects.  These can be removed from the SDDL string.